docs/pages/clients/compatibility.md

```diff
@@ -2,20 +2,18 @@
 title: Compatibility
 ---
 
-Expo Development Client contains three packages, which have to be compatible with each other. To check which versions are compatible, check the tables below.
+Expo Development Client is only compatible with specific versions of certain Expo and React Native modules.
```

Comment:
If someone detects an incompatibility that isn't on this list, we'd like for them to submit an issue